fluentd/run.sh

```diff
@@ -366,6 +364,10 @@ fi
 mkdir -p /var/log/fluentd/
 
 issue_deprecation_warnings
+# Output logs via stdout
+if [ -f $LOGGING_FILE_PATH ]; then
+  tail -n 1000 $LOGGING_FILE_PATH
+fi
```

Comment:
There probably isn't a log at this point - fluentd will create this log file

Reply:
Thanks, @richm.
Yeah, I also thought it cannot be this simple... :p  I verified the above `tail` just outputs the content of the log file at the startup...  Reverted it.

I wish "oc logs" took an option to tail the file or another option to pass a command line to run inside the pod...  but it seems there's on such trick (probably, it'd add a security issue, though...)

Reply:
not sure what you are trying to do here